client/my-sites/upgrade-nudge/expanded/index.jsx

```diff
@@ -0,0 +1,148 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import classNames from 'classnames';
+import page from 'page';
+
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import QueryPlans from 'components/data/query-plans';
+import Gridicon from 'components/gridicon';
+import PlanCompareCard from 'my-sites/plan-compare-card';
+import PlanCompareCardItem from 'my-sites/plan-compare-card/item';
+import TrackComponentView from 'lib/analytics/track-component-view';
+import formatCurrency from 'lib/format-currency';
+import { preventWidows } from 'lib/formatting';
+import { getFeatureTitle } from 'lib/plans';
+import { getPlanBySlug } from 'state/plans/selectors';
+import { PLAN_PERSONAL, getPlanClass, plansList } from 'lib/plans/constants';
+import analytics from 'lib/analytics';
+import { getCurrentPlan } from 'state/sites/plans/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+
+const ExpandedUpgradeNudge = ( {
```

Comment:
I think this component is quite huge and would actually benefit from being written as stateful/`class` one.